packages/lab/src/stepper-input/internal/useSpinner.ts

```diff
@@ -1,10 +1,12 @@
-import { useEffect, useState } from "react";
-
+import { useEffect, useState, KeyboardEvent, SyntheticEvent } from "react";
```

Comment:
KeyboardEvent doesnt seem to be used
```suggestion
import { useEffect, useState, SyntheticEvent } from "react";
```